doc/doxygen/headers/glossary.h

```diff
@@ -1485,7 +1485,16 @@
  *   paper lays out the motivation for this pattern and then proposes different ways
  *   of implementing it. It also compares the performance of different implementations.
  *
- *   The paper is currently in preparation.
+ * The full reference for this paper is as follows:
+ * @code
+@Article{TKB16,
```

Comment:
I just copy-pasted from other places in the file where we do the same. I don't think I care very much one way or the other.